pytorch_pfn_extras/runtime/_runtime.py

```diff
@@ -1,7 +1,20 @@
+from typing import (
+    Any, Dict, Generator, List, Optional, Sequence, Tuple, Union, TYPE_CHECKING
+)
+
 import torch
 
 _RUNTIME_TAG_NAME = '_ppe_runtime'
 
+DeviceLike = Union[str, torch.device]
+Batch = Union[torch.Tensor, List[torch.Tensor],
+              Tuple[torch.Tensor, ...], Dict[str, torch.Tensor]]
+Loader = Sequence[Batch]
```

Comment:
I think this should be `Iterable` as `torch.utils.data.DataLoader` is not a sequence.

Reply:
Agree